hydroflow_lang/src/graph/ops/lattice_join.rs

```diff
@@ -2,18 +2,18 @@ use quote::{quote_spanned, ToTokens};
 use syn::parse_quote;
 
 use super::{
-    FlowProperties, FlowPropertyVal, OperatorCategory, OperatorConstraints, OperatorWriteOutput,
-    Persistence, WriteContextArgs, RANGE_1,
+    DelayType, FlowProperties, FlowPropertyVal, OperatorCategory, OperatorConstraints,
+    OperatorWriteOutput, Persistence, WriteContextArgs, RANGE_1,
 };
 use crate::diagnostic::{Diagnostic, Level};
 use crate::graph::{OpInstGenerics, OperatorInstance};
 
 /// > 2 input streams of type <(K, V1)> and <(K, V2)>, 1 output stream of type <(K, (V1, V2))>
 ///
-/// Performs a group-by with lattice-merge aggregate function on LHS and RHS inputs and then forms the
-/// equijoin of the tuples in the input streams by their first (key) attribute. Note that the result nests the 2nd input field (values) into a tuple in the 2nd output field.
+/// Performs a reduce_keyed with lattice-merge aggregate function on each input and then forms the
```

Comment:
it is `fold_keyed` because the input type stream type and accumulator type don't have to be the same.